pawno/include/PPC_PlayerCommands.inc

```diff
@@ -2821,26 +2821,52 @@ COMMAND:convoymembers(playerid, params[])
 	return 1;
 }
 
-// Displays the statictics of the player
+/ Displays the statictics of the player
 COMMAND:stats(playerid, params[])
 {
 	// Setup local variables
-	new OtherPlayer;
+	new StatsMsg[1000], TitleMsg[128];
```

Comment:
You are too fast for me! :rofl: I forgot to define StatsPlayer here and had to change the snippet. Please apply the changes from https://gist.github.com/CodeAlDente/755b9a3216a9b9a892fd578da019a667. Thanks!